templates/Web/_composition/LaunchScript/Launch.AngularNode.AddScript/package$Launch.AngularNode.AddScript_gpostaction.json

```diff
@@ -0,0 +1,5 @@
+  "scripts": {
+//{[{
+    "start": "concurrently \"ng serve --port=3000 --o --proxy-config proxy.config.json\" \"nodemon server/server.js\"",
```

Comment:
Not super familiar with the templating system, but would it be possible to do this instead:

```
"start": "concurrently {[{\"ng serve --port=3000 --o --proxy-config proxy.config.json\"}]}",
```

Basically, decoupling frontend from backend scripts and thus reducing the number of `Launch.<Front><Back>.AddScript` templates?

Reply:
Honestly I wish this was possible as well. Right now at least to my knowledge, Inline merging is not possible.

Reply:
True decoupling would be as you suggested with something like  {[{\"ng serve --port=3000 --o --proxy-config proxy.config.json\"}]} however with the current capabilities of the core I don't think this is possible.

Actually I think I have an idea for something that might be possible I'm going to try it out.

Reply:
What if we had something like this?

```
    "start": "concurrently \"npm run start-frontend || yarn start-frontend\" \"npm run start-backend || yarn start-backend\"",
    "start-backend": "nodemon server/server.js",
    "start-frontend": "react-scripts start",
```

Reply:
This way I could merge in the start-frontend and start-backend scripts.

Reply:
That works actually! Nice one

Reply:
@SahilTara are you gonna update your PR with the code you guys mentioned in the conversation or you are going to stick with your original change?

Reviewed the original change, I am able to start my generate templates

Reply:
Yeah will be implementing these changes, have to do an assignment first though

Reply:
Hey @jcoc611-microsoft I have implemented the changes we discussed.